am6uno/Algorithm-Run-Time-Complexity
# Account creation

## Documentation
- Added a page after the keycloak login for the last steps in the account creation.
- Hid the navbar when the user is not signed in.
- Changed the userService.user object to hold the teacher or student object of the current user.
- Added postman collections for the student and teacher services.

## Testing
- update your .env file with the values from the .env.example file.
- run `docker compose down`
- run `mvn clean install`
- run `docker compose build`
- run `docker compose up` or `docker compose -d`
- After the containers are up run `mvn spring-boot: run`
- Go to ui/encounter and run `npm start`
- Try creating both a student and teacher account and verify both are in the database by using postman.
- Try logging into the accounts you've created and verify that it signs you in.
